### PR TITLE
[FR] Update `People/Users_with_unique_titles`

### PR DESCRIPTION
**Pull request by Biokax**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Users_with_unique_titles/fr.md [hunk 1]
```diff
@@ -287,7 +282,7 @@ Ces joueurs ont obtenu leur titre respectif soit en réalisant un exploit de jeu
 | :-- | :-- | :-- |
 | ![][flag_NO] [-GN](https://osu.ppy.sh/users/895581) | Champion Above Champions | A réalisé le premier FC NoMod sur [Shotgun Symphony+ [Impossibly Intense]](https://osu.ppy.sh/beatmapsets/7671#osu/32570) |
 | ![][flag_AU] [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | A réalisé un FC sur toutes les beatmaps classées du mode osu!taiko le 3 novembre 2019. |
-| ![][flag_NL] [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | *Non documenté* |
+| ![][flag_NL] [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | Décerné par ![][flag_CA] [awp](https://osu.ppy.sh/users/2650) à la fin de l'année 2008 pour avoir atteint le rang #104 en une semaine, en tant que référence à [*The World Ends with You*](https://fr.wikipedia.org/wiki/The_World_Ends_with_You) <!-- source: https://fr.wikipedia.org/wiki/The_World_Ends_with_You (Shirotora_old's forum signature) --> |
```
[the remark this answers is not in the document]
Comment (Biokax) [reply wiki/People/Users_with_unique_titles/fr.md:285]: why delete the comment ? its the same as the en.md one

[the remark this answers is not in the document]
Comment (DoryanR) [reply wiki/People/Users_with_unique_titles/fr.md:285]: Franchement j'en vois pas l'utilité sur les traductions, je retire souvent ces notes sur les traductions, alors autant uniformiser le tout

[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/People/Users_with_unique_titles/fr.md]: like all comments, they're for wiki editors, not the readers. there's isn't really any use case for having them in translations, so it's ok to remove

File: wiki/People/Users_with_unique_titles/fr.md [hunk 2]
```diff
@@ -287,7 +282,7 @@ Ces joueurs ont obtenu leur titre respectif soit en réalisant un exploit de jeu
 | :-- | :-- | :-- |
 | ![][flag_NO] [-GN](https://osu.ppy.sh/users/895581) | Champion Above Champions | A réalisé le premier FC NoMod sur [Shotgun Symphony+ [Impossibly Intense]](https://osu.ppy.sh/beatmapsets/7671#osu/32570) |
 | ![][flag_AU] [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | A réalisé un FC sur toutes les beatmaps classées du mode osu!taiko le 3 novembre 2019. |
-| ![][flag_NL] [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | *Non documenté* |
+| ![][flag_NL] [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | Décerné par ![][flag_CA] [awp](https://osu.ppy.sh/users/2650) à la fin de l'année 2008 pour avoir atteint le rang #104 en une semaine, en tant que référence à [*The World Ends with You*](https://fr.wikipedia.org/wiki/The_World_Ends_with_You)
```
Comment (Biokax) [inline wiki/People/Users_with_unique_titles/fr.md:285]:
```suggestion
| ![][flag_NL] [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | Décerné par ![][flag_CA] [awp](https://osu.ppy.sh/users/2650) à la fin de l'année 2008 pour avoir atteint le rang #104 en une semaine, en tant que référence à [*The World Ends with You*](https://fr.wikipedia.org/wiki/The_World_Ends_with_You) |
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**